tool/prdeployer.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2017, FaisalAbid. All rights reserved. Use of this source code
```

Comment:
The AUTHORS file exists, it's in the root of the repo. The copyright on every other file should be identical (that's what the CLA you signed is about :-) ).